src/main/steps/respondent/hub-page/content.ts

```diff
@@ -322,13 +326,18 @@ const cy: typeof en = ({ isDivorce, partner, userCase, contactEmail }: CommonCon
     line1: `Your ${partner} has applied for a ‘final order’. The application will be checked by court staff. If there are no other applications that need to be completed then your ${
       isDivorce ? 'divorce will be finalised' : 'civil partnership will be legally ended'
     }.`,
-    line2: `${
+    line2: `You need to pay ${getFee(
+      config.get('fees.finalOrderApplicationFee')
+    )} for the application before it can be submitted. Phone 0300 123 1711 to make payment. Have your card details ready.`,
+    line3:
+      'If you need help paying the fee then you will need to apply for Help With Fees first. Then phone with your Help With Fees reference' +
+      ' number. You can ',
+    line4: `${
       dayjs().isAfter(userCase.dateFinalOrderNoLongerEligible)
         ? `You will receive an email by ${getFormattedDate(
-            dayjs(userCase.dateFinalOrderSubmitted).add(config.get('dates.finalOrderSubmittedOffsetDays'), 'day'),
-            SupportedLanguages.Cy
+            dayjs(userCase.dateFinalOrderSubmitted).add(config.get('dates.finalOrderSubmittedOffsetDays'), 'day')
```

Comment:
if this is for the welsh property, worth adding the second argument (SupportedLanguages.Cy) in the function to get the Welsh text for the date format.